ci/pipeline

```diff
@@ -22,11 +22,13 @@ val PACKAGE_DOCS_DIR: Path = pwd / 'target / "universal-docs"
 
 /**
  * Compile Marathon and run unit and integration tests followed by scapegoat.
+ *
+ * @param logFileName Name of file which collects logs.
```

Comment:
please add: `Default value exits to be able to run this target locally`

Reply:
I've removed the default because it led to bugs.